modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,246 @@
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only | Boolean | Whether this source should only be executed on the primary node when running in cluster. |  | 
+| Streaming Strategy | 
+* xref:apikit-graphql-module-reference.adoc#repeatable-in-memory-stream[Repeatable In Memory Stream] 
+* xref:apikit-graphql-module-reference.adoc#repeatable-file-store-stream[Repeatable File Store Stream]
+* Non-Repeatable Stream | Configure if repeatable streams should be used and their behavior. |  | 
+| Redelivery Policy | xref:apikit-graphql-module-reference.adoc#redelivery-policy[Redelivery Policy] | Defines a policy for processing the redelivery of the same message. |  | 
+| Object Type Name | String | The object-type name |  | *x*
+| Field Name | String | The field name. |  | 
+| Body Content | Any | Content of the message (usually, the payload). | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+| *Attributes Type* | * xref:apikit-graphql-module-reference.adoc#data-fetcher-attributes[Data Fetcher Attributes] 
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config] 
+
+=== Data Loader
+
+`<graphql-router:data-loader>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Object Type Name | String | The object-type name. |  | *x*
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only | Boolean | Whether this source should only be executed on the primary node when running in cluster. |  | 
+| Redelivery Policy | xref:apikit-graphql-module-reference.adoc#redelivery-policy[Redelivery Policy] | Defines a policy for processing the redelivery of the same message. |  | 
+| Body Content | Any | Content of the message (usually, the payload). | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Array of Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config] 
+
+== Types
+
+=== Data Fetcher Attributes
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Alias | String | Alias of the queried field. |  | false
+| Arguments | Object | Arguments of the queried field. |  | false
+| Selected Fields | Object | Map of subfields selected from this compound type. |  | 
+| Source | Any | Actual value of the fetched field parent. |  | false
+|======================
+
+=== Repeatable In Memory Stream
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Initial Buffer Size | Number | This is the amount of memory that is allocated to consume the stream and provide random access to it. If the stream contains more data than buffer capacity, then it is expanded according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  | 
```

Comment:
Previously, the descriptions don't have "This is".

I don't think we're super clear on whether to put attributes, etc, in mono font in tables, so I guess this is fine.

Reply:
I'll remove the "This is" and make some attributes, etc in monofont. Thanks!